examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/controlexample/TableTab.java

```diff
@@ -258,6 +271,75 @@ void createOtherGroup () {
 		resizableColumns.addSelectionListener (widgetSelectedAdapter(event -> setColumnsResizable ()));
 		headerImagesButton.addSelectionListener (widgetSelectedAdapter(event -> recreateExampleWidgets ()));
 		subImagesButton.addSelectionListener (widgetSelectedAdapter(event -> recreateExampleWidgets ()));
+		editableButton.addSelectionListener(widgetSelectedAdapter(event -> makeTableContentEditable()));
+	}
+
+	/**
+	 * Make Tree Editable
+	 */
+	void makeTableContentEditable() {
```

Comment:
Please transfer the comments I made for the `TreeTab` to this class as well (if they apply here). In addition, the comment is inconsistent (probably due to copy/paste), but since it is obsolete, you can simply remove it.